kura/org.eclipse.kura.api/META-INF/MANIFEST.MF

```diff
@@ -29,13 +29,13 @@ Export-Package: org.eclipse.kura;version="1.3.0",
  org.eclipse.kura.gpio;version="1.0.1",
  org.eclipse.kura.linux.udev;version="1.0.0",
  org.eclipse.kura.message;version="1.1.0",
- org.eclipse.kura.net;version="1.2.1",
+ org.eclipse.kura.net;version="1.2.2",
```

Comment:
So in the past we the argument over and over that Kura follows semantic versioning ... now looking at the suggestions from "bndtools" about the API version from Kura 2.1.0 and this PR branch it shows:

```
===============================================================
* org.eclipse.kura.api 1.2.0.201702240825-1.1.0 suggests 2.0.0
===============================================================
  Package                                            Delta      New        Old        Suggest    If Prov.  
* org.eclipse.kura                                   MAJOR      1.3.0      1.2.1      2.0.0      -         
  org.eclipse.kura.annotation                        ADDED      1.0.0      -          ok         -         
  org.eclipse.kura.asset                             ADDED      1.0.0      -          ok         -         
  org.eclipse.kura.asset.listener                    ADDED      1.0.0      -          ok         -         
* org.eclipse.kura.cloud                             MAJOR      1.1.0      1.0.1      2.0.0      1.0.1     
  org.eclipse.kura.driver                            ADDED      1.0.0      -          ok         -         
  org.eclipse.kura.driver.listener                   ADDED      1.0.0      -          ok         -         
* org.eclipse.kura.gpio                              MAJOR      1.0.1      1.0.1      2.0.0      -         
* org.eclipse.kura.net                               MAJOR      1.3.0      1.2.1      2.0.0      1.2.1     
* org.eclipse.kura.net.wifi                          MAJOR      1.2.1      1.2.0      2.0.0      -         
* org.eclipse.kura.system                            MINOR      1.1.0      1.1.0      1.2.0      -         
  org.eclipse.kura.type                              ADDED      1.0.0      -          ok         -         
  org.eclipse.kura.wire                              ADDED      1.0.0      -          ok         -         
```

So the suggestion is to move to 2.0.0, which is what @kartben suggested. And I agree.

Unless of course you want to give up semantic versioning …